modules/n1ql/pages/n1ql-language-reference/createsequence.adoc

```diff
@@ -0,0 +1,414 @@
+
+bucket::
+(Optional) The bucket in which you want to create the sequence.
+
+scope::
+(Optional) The scope in which you want to create the sequence.
+
+identifier::
+(Required) The name of the sequence.
+The sequence name must be unique within the scope.
+
+Currently, only the `default` namespace is available.
+If you omit the namespace, the default namespace in the current session is used.
+
+If the {query-context}[query context] is set, you can omit the bucket and scope from the statement.
+In this case, the bucket and scope for the sequence are taken from the query context.
+
+The namespace, bucket, scope, and sequence name must follow the rules for {identifiers}[identifiers].
+If the namespace, bucket, scope, or sequence name contain any special characters such as hyphens (-), you must wrap that part of the expression in backticks ({backtick} {backtick}).
+
+[[if-not-exists]]
+=== IF NOT EXISTS Clause
+
+The optional `IF NOT EXISTS` clause enables the statement to complete successfully when the specified sequence already exists.
+If a sequence with the same name already exists within the specified scope, then:
+
+* If this clause is not present, an error is generated.
+
+* If this clause is present, the statement does nothing and completes without error.
+
+[[create-sequence-options]]
+=== Sequence Options
+
+[source,ebnf]
+----
+include::{partialsdir}/grammar/ddl.ebnf[tag=create-sequence-options]
+----
+
+image::n1ql-language-reference/create-sequence-options.png["Syntax diagram: refer to source code listing", align=left]
+
+You can use the following optional clauses to specify individual attributes for the sequence.
+These clauses can occur in any order, but none of them can occur more than once in the statement.
+
+[horizontal.compact]
+start-with:: <<start-with>> icon:caret-down[]
+increment-by:: <<increment-by>> icon:caret-down[]
+maxvalue:: <<maxvalue>> icon:caret-down[]
+minvalue:: <<minvalue>> icon:caret-down[]
+cycle:: <<cycle>> icon:caret-down[]
+cache:: <<cache>> icon:caret-down[]
+
+[[start-with]]
+==== START WITH Clause
+
+[source,ebnf]
+----
+include::{partialsdir}/grammar/ddl.ebnf[tag=start-with]
+----
+
+image::n1ql-language-reference/start-with.png["Syntax diagram: refer to source code listing", align=left]
+
+Use the START WITH clause to specify the starting value for the sequence.
+
+If this clause is omitted, the default start value is `0`.
+
+[horizontal#start-with-args]
+integer::
+(Required) The starting value for the sequence.
+
+[[increment-by]]
+==== INCREMENT BY Clause
+
+[source,ebnf]
+----
+include::{partialsdir}/grammar/ddl.ebnf[tag=increment-by]
+----
+
+image::n1ql-language-reference/increment-by.png["Syntax diagram: refer to source code listing", align=left]
+
+Use the INCREMENT BY clause to specify the increment value of each step in the sequence.
+
+If this clause is omitted, the increment value is `1`: that is, at each step in the sequence, the value goes up by `1`.
```

Comment:
nit: I find it hard to see wheterh the colon is part of the rendered highlighting or not. Break into two sentences instead? 
...`1`. That is,...